dataflow_test.py

```diff
@@ -513,6 +513,11 @@ def test_pipes_must_end_in_a_sink():
 
 
 def test_count_filter():
+
+    # count_filter provides a future/filter pair.
+    # This is a simple interface to keep track of
+    # how many entries satisfy the predicate.
```

Comment:
Maybe mention that it not only keeps track of how many pass, but also how many were filtered out (which I had forgotten myself).